### ISSUE TITLE
Problem updating to Prestashop 8*

### ISSUE THREAD

**Issue by mriberavlt** [PrestaShop/PrestaShop#36693]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

I have Prestashop 178.11

In 1-click Update I get a message that the requirements are not met: Your current version of the module is out of date. Update now Modules &gt; Module Manager &gt; Updates.

Where it tells me that in those modules you have to have PHP 8.1, when I have 7.4.

But if I update PHP to 8, the page goes blank.

If I go back to PHP 7.4 the modules continue to tell me that they need PHP 8.1.

I can't update Prestashop to 8 at all

### Expected behavior

I hope to be able to update Prestashop!

### Steps to reproduce

1-Click Update > Your current version of the module is out of date. Update now [Modules &gt; Module Manager &gt; Updates](https://carrer.venivinum.cat/xxxxxx/index.php/improve/modules/updates?_token=VwaJU33fIoNn-44Wz4fYYjFlFBq52szIANKKTTAmuYI)

### PrestaShop version(s) where the bug happened

1.7.8.11

### PHP version(s) where the bug happened

8.0

### If your bug is related to a module, specify its name and its version

MUltiple like: Best Costumers, payment with transaccion, best ctergories

### Your company or customer's name goes here (if applicable).

carrer.venivum.cat

**Comment by M0rgan01** [comment]
Hello @mriberavlt 

Could you provide screenshots of your errors? Or logs? This would allow us to better understand your error and thus be able to help you.

**Comment by AureRita** [comment]
Hi @mriberavlt  ,

Since we had no news from you for more than 30 days, I'll close this ticket to avoid cluttering up the backlog. Please note that you can always create a new one if further information pops up.

Thanks

**Comment by mriberavlt** [comment]
Hi,

I couldn't because the website was live and seeing that I couldn't change
the theme, I tried to go back to 1.7.8 and, as always, I got a white screen
both on the website and back.

And although I can make backups from the provider's website, none of them
worked

And I had to make a manual backup using FTP
Salut!

Marc


Missatge de Morgan Pichat ***@***.***> del dia dl., 14 d’oct.
2024 a les 10:06:
